appveyor.yml

```diff
@@ -0,0 +1,39 @@
+init:
+  - git config --global core.autocrlf false
+
+clone_depth: 5
+
+environment:
+  access_token:
+    secure: VMFbecLLHzDq/09YDPbcM0VDDSwwgY57vr5GXK6cZZ4Ti/Xs5RZoylzV8MMr1350
+
+install:
+  - python -m pip install -r requirements.txt
```

Comment:
if the environment doesn't get reset each time, you may want to run this install in a virtual environment.  That will cover the case when a module is removed